Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/CustomInputSelector.cs

```diff
@@ -51,13 +46,8 @@ private enum InputSourceNumber
 
         private void Awake()
         {
-            bool spawnControllers = false;
+            bool spawnControllers = Application.isEditor && simulateHandsInEditor;
```

Comment:
The old way took into account whether a headset is present. Otherwise, I'll get the little hands stuck in my view inside the headset when running via the editor.

Reply:
Did it though?  I think the latest versions of the Editor will now return that the device is present even when it's unplugged.

Reply:
I'm not able to repro that in 2017.2 or 2017.3

![image](https://user-images.githubusercontent.com/3580640/34426793-0cfb908e-ebef-11e7-9396-2e93b9caf5a9.png)

https://github.com/Microsoft/MixedRealityToolkit-Unity/issues/1537#issuecomment-354371764